analysis_driver/reader/demultiplexing_parsers.py

```diff
@@ -2,139 +2,146 @@
 from itertools import islice
```

Comment:
This and `ElementTree` are no longer used and can be removed

Reply:
Removed